### PR TITLE
Add AV1 decoder to Chrome 70

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This CL adds AV1 decoder to Audio/Video updates in Chrome 70 /updates article and updates previous one.

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
@petele Any idea what is "ERRORS
src/content/en/updates/2018/09/chrome-70-media-updates.md#L32 - Link contains unescaped whitespace or accidental line break."?

File: src/content/en/updates/2018/09/chrome-70-media-updates.md [hunk 1]
```diff
@@ -2,23 +2,52 @@ project_path: /web/_project.yaml
...
+Desktop x86-64 based on the [official bitstream specification]. At this time,
+support is limited to “Main” [profile 0] and does not include encoding
+capabilities. The supported container is [MP4 (ISO-BMFF)] (see [From raw video
+to web ready] for a brief explanation of containers).
+
+To try AV1:
+
+- Go to the [YouTube TestTube page].
+- Select "Prefer AV1 for SD" or "Always Prefer AV1" to get the desired
+  resolution AV1. Note that at higher resolutions, AV1 is more likely to
```
Comment (jpmedley) [inline src/content/en/updates/2018/09/chrome-70-media-updates.md]: 'AV1 resolution', not 'resolution AV1'.

File: src/content/en/updates/2018/09/chrome-70-media-updates.md [hunk 2]
```diff
@@ -2,23 +2,52 @@ project_path: /web/_project.yaml
...
+  experience playback performance issues on some devices.
+- Try playing YouTube clips from the [AV1 Beta Launch Playlist].
+- Confirm the codec av01 in "Stats for nerds"
```
Comment (jpmedley) [inline src/content/en/updates/2018/09/chrome-70-media-updates.md]: Needs a period.

**Comment by jpmedley** [comment]
@beaufortfrancois @petele Francois, I'll be unavailable for a few days after 24:00 UTC. If you make these changes after that you'll need someone else to approve.

**Comment by beaufortfrancois** [comment]
@jpmedley I've addressed your feedback and fixed husky issue.

**Pull request merged**